rps.js

```diff
@@ -0,0 +1,64 @@
+const playGame = function (a) {
+  var win = " ";
+  let b;
```

Comment:
Make sure you always give proper names to your variables. `a` and `b` are meaningless.